superset-frontend/src/components/DatabaseSelector/DatabaseSelector.test.tsx

```diff
@@ -272,7 +272,6 @@ test('should display options in order of the api response', async () => {
 });
 
 test('Should fetch the search keyword when total count exceeds initial options', async () => {
-  fetchMock.reset();
```

Comment:
restore from [this accidental removal](https://github.com/apache/superset/pull/29118/files#diff-c37b4ee2b1c075411b41fa8bd037c97f211ef8b087ed331c6884d2634c3ca001R275)